.github/workflows/py_test.yml

```
@@ -0,0 +1,28 @@
+on:
```

```suggestion
name: Test Code

on:
```

To keep it in form with the lint yaml